src/components/CustomCollection/CustomCollection.tsx

```diff
@@ -261,7 +270,10 @@ const CustomCollection = ({
               <Button
                 type="button"
                 unstyled
-                onClick={() => setIsAddingLink(false)}>
+                onClick={() => {
+                  setIsAddingLink(false)
+                  setDisableDragAndDrop(false)
```

Comment:
Question: Is it possible to mock these out and check they are called in a unit test? I'm okay if not since there is an e2e test for this.

Reply:
I think I tried to write a test for `setIsAddingLink` a while back and found it to be very tricky, so not sure about that one. But `setDisableDragAndDrop` might be easier, `myspaceContext` is already mocked out. I'll try to find some time before EOD to put something up.